### PR TITLE
 About-us page

### PR DESCRIPTION
**Pull request by Davidsonabimbola**
The about us page was designed into sections:
Our mission
Our Approach

### CODE REVIEW

File: src/Pages/About_us.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import Rectangle64 from "../img/Rectangle64.png";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const About_us: React.FC = () => {
+  return (
+    <div>
+    
+        <h2 className="text-center text-success  mb-5 mt-3" >About us</h2>
+        <div> 
+
```
Comment (Oluyide) [inline src/Pages/About_us.tsx:15]: Remove this space please

File: src/Pages/About_us.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      <Col sm={6}>
+          <h2 className="text-center">Our Mission</h2>
+          <p><span></span></p>
+          <div>Our mission is to help businesses solve problems by building affordable IT solutions to make their processes seamless and achieve their organisational objectives.</div>
+      </Col>
+      <Col></Col>
+    </Row>
+    </div>
+
+
```
Comment (Oluyide) [inline src/Pages/About_us.tsx:27]: Remove space

File: src/Pages/About_us.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          <div>We maintain a high recruitment standard and have a team of solution-oriented professionals ready to transform your business with the right product to suit your needs.</div>
+      </Col>
+    </Row>
+    </div>
+<br></br>
+<br></br>
+<br></br>
+    <div>
+    
+
```
Comment (Oluyide) [inline src/Pages/About_us.tsx:47]: Please remove this

File: src/Pages/About_us.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      <Col sm={6}>
+          <h2 className="text-center  mb-5">Our Process</h2>
+          <p><span></span></p>
+          <div>We operate an agile software development process that involves our world-class team of software developers tailoring our products to suit your specific business realities rather than offering a one size fits all solution.</div>
+      </Col>
+      <Col sm={6}><img src={Rectangle64} alt="Rectangle64"  width="500" height="300" className="rounded"/></Col>
+    </Row>
+    </div>
+    </div>
+
```
Comment (Oluyide) [inline src/Pages/About_us.tsx:58]: Please remove this, excess space

File: src/Pages/About_us.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+import Col from "react-bootstrap/Col";
+import Rectangle62 from "../img/Rectangle62.png";
+import Rectangle63 from "../img/Rectangle63.png";
+import Rectangle64 from "../img/Rectangle64.png";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const About_us: React.FC = () => {
+  return (
+    <div>
+
```
Comment (Oluyide) [inline src/Pages/About_us.tsx:12]: Why are alot of spaces everywhere

File: src/Footer.tsx [hunk 6]
```diff
@@ -21,10 +21,33 @@ const Footer: React.FC = () => {
...
-      <Col sm={3} className="text-center "></Col>
-      <Col sm={3} className="text-center"></Col>
-      <Col sm={3} className="text-center"><Facebook/><Twitter/><Youtube/></Col>
+      <Col lg={3} className="text-center text-white" >
+        <h5>Doyenify</h5>
+      <div>
+      61 community road,
+      <div>Akoka,yaba,Lagos.</div>
+        
+        </div>
```
Comment (OluyideO) [inline src/Footer.tsx:30]: Please remove the Excess spaces

File: src/Pages/About_us.tsx (hunk 1 above)
Comment (Davidsonabimbola) [reply src/Pages/About_us.tsx]: the spaces have been removed

File: src/Pages/About_us.tsx (hunk 2 above)
Comment (Davidsonabimbola) [reply src/Pages/About_us.tsx]: the spaces have been removed

File: src/Pages/About_us.tsx (hunk 3 above)
Comment (Davidsonabimbola) [reply src/Pages/About_us.tsx]: the spaces have been removed

File: src/Pages/About_us.tsx (hunk 5 above)
Comment (Davidsonabimbola) [reply src/Pages/About_us.tsx]: the spaces have been removed

File: src/Footer.tsx (hunk 6 above)
Comment (Davidsonabimbola) [reply src/Footer.tsx:30]: the spaces have been removed

**Review by OluyideO — APPROVED** [review]

**Review by TecFesco — APPROVED** [review]
good job

**Pull request merged**